TechnologyAssessmentApp/TechnologyAssessmentApp/Controllers/MasterTableViewController.swift

```diff
@@ -64,139 +62,67 @@ class MasterTableViewController: UITableViewController {
-            splitViewController?.showDetailViewController(detailNavigationController, sender: nil)
-        }
-    }
-    /*
-    // Override to support conditional editing of the table view.
-    override func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
-        // Return false if you do not want the specified item to be editable.
-        return true
     }
-    */
-
-    /*
-    // Override to support editing the table view.
-    override func tableView(_ tableView: UITableView, commit editingStyle:
-     UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
-        if editingStyle == .delete {
-            // Delete the row from the data source
-            tableView.deleteRows(at: [indexPath], with: .fade)
-        } else if editingStyle == .insert {
-            // Create a new instance of the
-            //appropriate class, insert it into the array, and add a new row to the table view
-        }    
-    }
-    */
-
-    /*
-    // Override to support rearranging the table view.
-    override func tableView(_ tableView: UITableView, moveRowAt fromIndexPath: IndexPath, to: IndexPath) {
-
-    }
-    */
-
-    /*
-    // Override to support conditional rearranging of the table view.
-    override func tableView(_ tableView: UITableView, canMoveRowAt indexPath: IndexPath) -> Bool {
-        // Return false if you do not want the item to be re-orderable.
-        return true
-    }
-    */
-    // MARK: - Navigation
-
-    // In a storyboard-based application, you will often want to do a little preparation before navigation
-//    override func prepare(for segue: UIStoryboardSegue, sender: Any?)
-//    {
-//        // Get the new view controller using segue.destinationViewController.
-//        // Pass the selected object to the new view controller.
-//
-//        guard let detailController = segue.destination as? DetailViewController
-//        else {
-//
-//            fatalError("Expected viewcontroller")
-//        }
-//
-//        detailController.
-//
-//    }
 }
-
 extension MasterTableViewController: UISplitViewControllerDelegate {
     func splitViewController(_ splitViewController: UISplitViewController,
          collapseSecondary secondaryViewController: UIViewController,
          onto primaryViewController: UIViewController) -> Bool {
                                 return isDetailViewController
                             }
 }
+
+// MARK: - Data Source
+class ArticleDataSource: CollectionArrayDataSource<Result, ArticleTableViewCell> {}
+
+// MARK: - Private Methods
+fileprivate extension MasterTableViewController {
+    func setUpDataSource() -> ArticleDataSource? {
+
+        if let articleModel = viewModel {
+            let dataSource = ArticleDataSource(tableView: tableView, array: articleModel.results)
+            dataSource.tableRowSelectionHandler = { [weak self] indexpath in
+                guard  let strongSelf = self else { return }
```

Comment:
identation is wrong, there's 2 spaces between `guard` and `let`. This should be fixed by running swiftlint.